.github/workflows/release.yml

```diff
@@ -103,6 +103,7 @@ jobs:
       with:
         name: ${{ matrix.language  }}-sdk.tar.gz
         path: ${{ github.workspace}}/sdk/${{ matrix.language }}.tar.gz
+      retention-days: 30
```

Comment:
There's no equivalent of the previous `skip-tags: true` here. Doesn't that mean we'll lose release artifacts after 30 days?

Reply:
Yes, it would! Sounds like I was too eager here, and we should not add this to the release or prerelease workflows, using the default instead.

Reply:
I think I saw you mentioned somewhere that 30 is an override of the default value which is 90. Don't we have the same problem after 90 days currently?

Reply:
@mikhailshilkov: This is the current status quo as far as I'm aware. [Here is a release run of 2 months ago with existing artifacts](https://github.com/pulumi/pulumi-aws/actions/runs/4898369557). Compare it to [this release of 3 months ago](https://github.com/pulumi/pulumi-aws/actions/runs/4823795386) - the artifacts are `Expired`.
If we want to keep them around for longer, we would have to come up with a new idea, but I'm unaware of any need for that.